shap/explainers/_deep/deep_tf.py

```diff
@@ -209,27 +246,35 @@ def _init_between_tensors(self, out_op, model_inputs):
             self.used_types[op.type] = True
 
     def _variable_inputs(self, op):
-        """ Return which inputs of this operation are variable (i.e. depend on the model inputs).
-        """
+        """Return which inputs of this operation are variable (i.e. depend on the model inputs)."""
         if op not in self._vinputs:
             out = np.zeros(len(op.inputs), dtype=np.bool)
-            for i,t in enumerate(op.inputs):
+            for i, t in enumerate(op.inputs):
                 out[i] = t.name in self.between_tensors
+                if not out[i]:
+                    if "lstm/while/lstm_cell" in t.name:
+                        out[i] = True
```

Comment:
This could be shortened into one line, but since the additivity doesn't fully hold I am keeping it here so its easier to debug via printing

Reply:
Can you explain the logic behind this change? It looks a bit like a hack to get around the fact that we no longer have self.between_tensors since we don't have the graph. Is that right?

Reply:
I actually posted an example explanation and deeper insights here: https://github.com/slundberg/shap/issues/930#issuecomment-700781695 . As stated there, without this SHAP assumes that all LSTM calculations do not impact the model output, thus the SHAP values are all zero. However, even with this "fix"
the results violate the additivity constraints, but since my matrix calculus is a bit rusty I was not able to figure out which gradients would need to be patched to fix this for good. As also explained, we do have the between tensors available 8see also comment), but the lstm_cells are not in them (for whatever reason).

Reply:
I actually posted an example explanation and deeper insights here: https://github.com/slundberg/shap/issues/930#issuecomment-700781695. 
As stated there, without this SHAP assumes that all LSTM calculations do not impact the model output, thus the SHAP values are all zero. However, even with this "fix" the results violate the additivity constraints, but since my matrix calculus is a bit rusty I was not able to figure out which gradients would need to be patched to fix this for good. As also explained, we do have the between tensors available (see also comment), but the lstm_cells are not in them (for whatever reason).